src/actions.py

```diff
@@ -20,7 +20,7 @@ def perform_operation(_action, target, self):
     elif operation == "cut" and self.get_mode() != 2:
         return
     args = []
-    if operation in ["center"]:
+    if operation in ("center"):
```

Comment:
Not something related to this PR directly, but now I'm seeing this I wonder if there's any reason why we're not just using `if operation == "center"`

Reply:
The reason for this was to keep it modular. In case other operations will get a dedicated setting, they should just be added to this list. This worked using list brackets but tuple brackets are ignored if a trailing comma isn't present. Fixed that now.

Reply:
Ah right, yeah that makes sense.